content/en/docs/papers/fsep/_index.md

```diff
@@ -0,0 +1,218 @@
+
+The first step in this process is to make defederated systems aware of the features provided by services such as The Bad Space, specifically its public search API and dynamically created and updated blocklists. 
+
+Federation is the exchange of messages through a common language, in this case, Activity Pub. A note is created using a specific format, and platforms that understand that format can read and respond to the original message. When this connection happens, sites are officially federated with each other.
+
+![federation process](federation-visual.png)
+
+Currently, federation can be prevented by adding a URL to the platform’s blocklist,  which will, in turn, refuse to accept activity from the site requesting communication, disallowing federation to happen.  
+
+
+While serviceable, this process has several challenges, such as: 
+
+- Exposing users to harmful content as means of filtering. 
+- Each problematic site must be manually added 
+- No mechanism for automation, which requires blocklists to be manually  maintained and updated 
+- No tool to validate connection requests at the user level for harmful content
+- Underdeveloped blocklist maintenance options to fine-tune what sites can communicate based on the administrator's discretion.
+
+Integration of The Bad Space’s features into backend functionality will improve  not only the administration experience but the individual users as well by: 
+
+- Allowing blocklists to be automatically imported during onboarding dramatically reduces the opportunity to be exposed to harmful content unnecessarily. 
+- The ability to request an updated blocklist or automate the process to check on its own periodically. 
+- Give the users the ability to vet incoming connection requests to validate that they are not from problematic sites. 
+- Expand blocklist management by listing why a site is blocked, access to available examples, and when the site was last updated.  
+
+While adding these features will enhance the usability of blocklists and upgrade the end-user experience, corresponding front-end changes are required in several areas to ask permission for these services to be used and to make how those services are being used clear and easy to understand.
+
+### Onboarding
+
+The administration onboarding process can be enhanced by leveraging the additions to the back end detailed in the Service Feature Integration. 
+
+| ID | FEATURE | PRIORITY |
+|:--|:--|:--|
+| 1.1 | At setup time, administrators should be able to upload a deny list or select a default to deny list provided. For MVP, only one deny list provider needs to be supported in this list (The Bad Space). Ultimately, the goal is to allow users to choose from one of several vetted tier 0 lists. | P1 |
+
+| 1.2 | At setup time, administrators should be able to select whether or not they want the default deny list to auto-update. If the admin selects “yes,” the list should attempt to auto-update once every 24 hrs.  | P1 |
```

Comment:
How does this relate to the PR on Mastodon regarding "configure blocklists at setup time" (not linking directly here, as to not draw attention to this before you're ready, but feel free to ask me for a link on discord)